packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -103,6 +110,226 @@ const en = {
     subtext: 'Restart the application after running the script',
     checkbox: 'I have already ran the command',
   },
+  initAccount: {
+    info: {
+      dialogBox: {
+        title: 'Add Coin/Account',
+        header: 'Follow instructions on the X1 Vault',
+        subheader: 'Add a coin/account Wallet',
+        subheader1: 'Cypherock Red',
+        dataArray: [
+          {
+            id: '1', // Add a unique identifier to each data object
+            leftImageSrc: bnbChainIcon,
+            text: 'BNB Chain 1',
+            rightImageSrc: checkIcon,
+          },
+          {
+            id: '2',
+            leftImageSrc: bitcoinIcon,
+            // rightText: '0.77 ETH',
+            text: 'Bitcoin 1',
+            rightImageSrc: halfLoaderGold,
+          },
+          {
+            id: '3',
+            leftImageSrc: etheriumBlueIcon,
+            text: 'Etherium 3',
+          },
+        ],
```

Comment:
The static lists should not be inside `en.ts`, It can be placed in the file where it's used. Since these are dummy variables which needs to be changed anyway

Reply:
AddAccount.ts file has been created. in store reducers has been created as well